packages/types/CHANGELOG.md

```diff
@@ -43,16 +47,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 - Re-release 0.2.1
 
-## [0.2.0] - 2022-12-06
-### Added
-- Support for `bypassBlocks` (#1435)
-
 ## [0.2.0] - 2022-11-17
 ### Changed
 - Sync with main sdk (#14)
 
+## [0.2.0] - 2022-12-06
+### Added
+- Support for `bypassBlocks` (#1435)
```

Comment:
Whats happened here? I think the version number might be wrong?

Reply:
fixed